docs/spark/tutorials/ml-sentment-analysis.md

```diff
@@ -0,0 +1,207 @@
+
+The following table contains sample data:
+
+|ReviewText|Sentiment|
+|-|-|
+|Wow... Loved this place.|    1|
+|Crust is not good.|    0|
+
+## Build your machine learning model
+
+1. Open Visual Studio and create a new C# Console App (.NET Core). Name the project *MLSparkModel*.
+
+1. In **Solution Explorer**, right-click the *MLSparkModel* project. Then select **Add > Machine Learning**.
+
+1. From the ML.NET Model Builder, select the **Sentiment Analysis** scenario tile.
+
+1. On the **Add data** page, upload the *yelptrain.csv* data set.
+
+1. Choose *Sentiment* from the **Columns to Predict** dropdown.
+
+1. On the **Train** page, set the time to train to *60 seconds* and select **Start training**. Notice the status of your training under **Progress**.
+
+1. Once Model Builder is finished training, **Evaluate** the training results. You can type phrases into the text box below **Try your model** and select **Predict** to see the output.
+
+1. Select **Code** and then select **Add Projects** to add the ML model to the solution.
+
+1. Notice that two projects are added to your solutions: **MLSparkModelML.ConsoleApp** and **MLSparkModelML.Model**.
+
+1. Double-click on your *MLSpark* C# project and notice that the following project reference has been added.
+
+   ```xml
+   <ItemGroup>
+       <ProjectReference Include="..\MLSparkModelML.Model\MLSparkModelML.Model.csproj" />
+   </ItemGroup>
+   ```
+
+## Create a console app
+
+Model Builder creates a console app for you.
+
+1. Right-click on **MLSparkModelML.Console** in Solution Explorer, and select **Manage NuGet Packages**.
+
+1. Search for **Microsoft.Spark** and install the package. **Microsoft.ML** is automatically installed for you by Model Builder.
+
+### Create a SparkSession
+
+1. Open the *Program.cs* file for **MLSparkModelML.ConsoleApp**. This file was autogenerated by Model Builder. Delete the `using` statements, the contents of the Main() method, and the `CreateSingleDataSample` region.
+
+1. Add the following additional `using` statements to the top of the *Program.cs*:
+
+   ```csharp
+   using System;
+   using System.Collections.Generic;
+   using Microsoft.ML;
+   using Microsoft.ML.Data;
+   using Microsoft.Spark.Sql;
+   using MLSparkModelML.Model;
+   ```
+
+1. Change the `DATA_FILEPATH` to the path of your *yelptest.csv*.
+
+1. Add the following code to your `Main` method to create a new `SparkSession`. The Spark Session is the entry point to programming Spark with the Dataset and DataFrame API.
+
+   ```csharp
+   SparkSession spark = SparkSession
+        .Builder()
+        .AppName(".NET for Apache Spark Sentiment Analysis")
+        .GetOrCreate();
+   ```
+
+   Calling the *spark* object created above allows you to access Spark and DataFrame functionality throughout your program.
+
+### Create a DataFrame and print to console
+
+Read in the Yelp review data from the *yelptest.csv* file as a `DataFrame`. Include `header` and `inferSchema` options.
```

Comment:
Do we want to explain why we're using `header` and `inferSchema` or what they do?